src/Container/GenericContainer.php

```diff
@@ -55,11 +63,11 @@ public function config(object $config): self
         return $this;
     }
 
-    public function get(string $className): object
+    public function get(string $className, mixed ...$params): object
     {
         $this->log->startResolving();
 
-        return $this->resolve($className);
+        return $this->resolve($className, ...$params);
```

Comment:
Added `...$params` support, not sure whether to keep it.

Reply:
Yes. 😉